slab/quill
# Fix white spaces not being preserved when pasted into editor

Previously, only the tag name is checked. However, browsers and applications (e.g. Google Docs) may add `white-space` style for elements that need to preserve whitespaces.

To test:

1. Type "hello     world" into Google Docs
2. Copy all text from Google Docs
3. Paste into a Quill editor
4. Make sure spaces between the two words are preserved
